### DISCUSSION TITLE
airsim swarm control

### DISCUSSION THREAD

**Discussion by LeiH5180** [microsoft/AirSim#4703] [General]
Can AIRSIM control multiple UAVs (eg:30)at the same time? 
Because when the number of UAVs exceeds 13, the flight of UAVs becomes slow

**Comment by xxEoD2242** [comment]
@LeiH5180 Our platform, [SWARM Simulation Platform](https://www.swarmsim.io) was built for controlling large numbers of agents in the cloud. You can check that out if you like.
